innopoints/views/file.py

```diff
@@ -82,10 +83,20 @@ def retrieve_file(file_id):
 def delete_file(file_id):
     """Delete the given file by ID."""
     file = StaticFile.query.get_or_404(file_id)
+    if file.owner != current_user:
```

Comment:
what about admins? they should be able to remove anything

Reply:
It's not like we have an interface to remove files so it doesn't really matter